### DISCUSSION TITLE
Pause buffering

### DISCUSSION THREAD

**Discussion by Maddylicious** [nirgoren/Eddienput#38] [Technical support]
I'm looking for a way to easily measure frame data in GGST, and I'd like a way to easily perform perfect pause buffering. Would that be possible using Eddienput?

**Comment by nirgoren** [comment]
I'm not sure what you mean by "perfect pause buffering". Do you mean something along "pause, un-pause for 1 frame, pause again"?
That would depend on whether the game allows un-pausing for a single frame before pausing again. If so I guess that would be possible, but you'd have to find out the time it takes between pressing start to close the pause menu and the menu actually closing for the game to resume.

Regardless, you can use two instances of Eddienput (toggle `suppress hotkeys` to take manual control over one controller at a time for setup) and have both P1 and P2 perform some action starting at the same frame. That could help with measuring frame data.

**Reply by Spabobin** [reply]
I tried running 2 instances (v2.0) to test some things in USF4 training mode. P1 was controlled with my stick and I used the "Home" key for P2 Start (which is what has always worked for testing training mode stuff), and using the same playback hotkey for both instances. I can get the playback working with either P1 or P2, but never both at once. Any idea what I could be doing wrong?

I was hoping to test armor frames by finding the exact timing where armor stops working, which would otherwise require a ton of trial and error. It could just be a game-specific thing with how controls are set up I guess.

Edit: maybe if I made the P2 instance use an alternative config json? I'll try messing around with it
